lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -47,7 +49,7 @@
 
   private static final Logger LOG = Logger.getLogger(HandlerContainer.class.getName());
 
-  private static final AtomicInteger COUNTER = new AtomicInteger(0);
+  private static final AtomicInteger counter = new AtomicInteger(0);
```

Comment:
why this change? it seems orthogonal to the needs of the JIRA. Also, I believe this should violate the checkstyle, but @tcNickolas knows best.